crates/runtime_isolate/Cargo.toml

```diff
@@ -15,10 +15,21 @@ log = { version = "0.4.18", features = ["std", "kv_unstable"] }
 once_cell = "1.17.1"
 async-recursion = "1.0.4"
 linked-hash-map = "0.5.6"
+flate2 = "1.0.26"
+serde_v8 = "0.100.0"
```

Comment:
For now, we're avoiding using `serde_v8`, but might switch in the future.

Reply:
I think serde_v8 is very useful. Do we need to remove it? It has simplified deserialization for me.

Reply:
Let's remove it for now, and we can still later switch to `serde_v8` on the whole application instead of using it in a single place (note that we'll probably have to benchmark it to make it doesn't slow things down too much)

Reply:
done